scripts/crowbar-testbuild.py

```diff
@@ -37,9 +37,20 @@
     'master':                'develcloud6'
 }
 
+MKCLOUD_HA_PARAMETERS = (
+    'nodenumber=4', 'hacloud=1',
+    'networkingplugin=vxlan',
+    'clusterconfig="data+services+network=2"')
+
+
+MKCLOUD_CEPH_PARAMETERS = (
+    'nodenumber=3', 'networkingplugin=linuxbridge')
```

Comment:
If possible, an explicit want_ceph=1 would be nice, since there's always the possibility that we might just decouple linuxbridge and ceph.